pkg/chaining/step.go

```diff
@@ -8,12 +8,13 @@ import (
 type Step struct {
 	Service service.Service
 	Name    string
+	Last    bool
 
 	params *configuration.JiraAPIResourceParameters
 }
 
-func (s *Step) Execute(csValues CrossStepsValues) error {
-	return service.Execute(s.Service, *s.params)
+func (s *Step) Execute(csValues CrossStepsValues, lastStep bool) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method Step.Execute should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)